### ISSUE TITLE
Bed / Nozzle Temps Reading Random Values

### ISSUE THREAD

**Issue by CollinCBoring** [makerbase-mks/MKS-Robin-Nano-V3.X#49]
Problem: I have fully wired in the Robin Nano V3 and I am getting temp readings that bounce between -8000 and 8000.  The values appear to be random and change quickly.  When I unplug the 100K thermistors from the board, the values go to zero. No other issues are present with controlling the movement of the machine, homing, or auto-bed leveling.

Tests I have Completed: 
1. Verified the 100K thermistor is functional and tried various new sensors.
2. Tested the firmware across other Robin Nano V3 boards I have.

Files Attached:
1. Marlin Firmware 2.0 from the Current link folder. https://github.com/makerbase-mks/Mks-Robin-Nano-Marlin2.0-Firmware
2. Pictures from my screen.

Configured for:
1. H43 Touchscreen
2. Sensorless Homing on X & Y
3. BL Touch
4. TMC2209 in UART Mode
5. Smart Filament Runout Sensor

[Marlin Firmware.zip](https://github.com/makerbase-mks/MKS-Robin-Nano-V3.X/files/6835619/Marlin.Firmware.zip)
![IMG_2627](https://user-images.githubusercontent.com/47790904/126054739-2797c888-8ff5-40ba-91a9-9dfd85d622a9.jpg)
![IMG_2628](https://user-images.githubusercontent.com/47790904/126054740-93368ee6-4964-4446-8a4a-a3d2abef2eb8.jpg)
![IMG_2629](https://user-images.githubusercontent.com/47790904/126054741-78d87cbc-b14a-4c90-be99-ca7a44494dc2.jpg)
![IMG_2630](https://user-images.githubusercontent.com/47790904/126054742-8dac6694-a9be-42ae-a77a-63291bb4cad6.jpg)
![IMG_2631](https://user-images.githubusercontent.com/47790904/126054743-697e507f-166b-45be-8fe1-85f4c6d74098.jpg)
![IMG_2632](https://user-images.githubusercontent.com/47790904/126054745-23f0be9c-6769-458d-b4a3-1ecd1afb5b83.jpg)

**Comment by mks-viva** [comment]
I will report this problem to the engineer, can you try to use the latest marlin firmware?

**Comment by CollinCBoring** [comment]
> I will report this problem to the engineer, can you try to use the latest marlin firmware?

Yes I will download the latest firmware today and see if it changes the readings. I will post the results within the next 24 hours.

**Comment by CollinCBoring** [comment]
I downloaded the latest version of Marlin 2,0 from the Makerbase Repository and I have the same issue. I did learn though when I configure the board for the TFT35 V2.0, the temp readings are normal. I tried flashing the H43 with NO BEEP from H43_Firmware_for_Malin and H43_Firmware_1.31, same issue with both.

Here are the steps I take to create the problem.
1. Download the latest version of Marlin 2.0 from the makerbase repository.
2.  Disable the following:
//#define MKS_TS35_V2_0
//#define TFT_LVGL_UI
//#define TOUCH_SCREEN
3. Enable the following:
#define DGUS_LCD_UI_MKS
4. Compile and load to Robin Nano V3 using flash drive
5. Update the H43 using the latest DWIN_SET.

**Comment by CollinCBoring** [comment]
Any progress?

**Comment by CollinCBoring** [comment]
Any Progress @makerbase-mks @MKS-Sean

**Comment by Grant09** [comment]
hi, we have the same issue, pls help

**Comment by slapukas** [comment]
hi, we have the same issue, pls help

**Comment by Grant09** [comment]
> hi, we have the same issue, pls help

Hey man send me a email at grant@cinch-it.co.za and I'll send you the files to fix jt

**Comment by CollinCBoring** [comment]
@MKS-Sean @makerbase-mks Any Progress Here???
